apps/cowswap-frontend/src/modules/bridge/pure/contents/BridgingProgressContent/ReceivedBridgingContent/index.tsx

```diff
@@ -70,8 +74,8 @@ export function ReceivedBridgingContent({
         </b>
       </ConfirmDetailsItem>
 
-      {depositLink && <TransactionLink link={depositLink} label="Source transaction" />}
-      {fillTxLink && <TransactionLink link={fillTxLink} label="Destination transaction" />}
+      {depositLink && <TransactionLink link={depositLink} label="Source transaction" chainId={sourceChainId} />}
+      {fillTxLink && <TransactionLink link={fillTxLink} label="Destination transaction" chainId={destinationChainId as SupportedChainId} />}
```

Comment:
This will break when we use OP for example.
But this would already break even without this change.

@elena-zh can you add to the list of issues to address this use case?

Reply:
You are right.
For unsupported networks we should use `useBridgeSupportedNetworks()` instead